platformio.ini

```diff
@@ -81,31 +81,33 @@ lib_ldf_mode = chain+
 lib_deps = 
   https://github.com/emsesp/EMS-ESP-Modules.git
 
+;
 ; build for GitHub Actions CI
+;
 ; the Web interface is built seperately, so is skipped in extra_scripts
-[env:ci]
-; 4MB using Tasmota (no SSL) - like BBQKees older S32
+;
+[env:ci_s_4M]
+; 4MB ESP32 - using Tasmota - no SSL, no PSRAM - like BBQKees older S32
 extends = espressi32_base_tasmota
 extra_scripts = scripts/rename_fw.py
 board = esp32dev
 board_build.partitions = esp32_partition_4M.csv
-board_build.extra_flags = -DBOARD_HAS_PSRAM
+build_flags =
+  ${espressi32_base_tasmota.build_flags}
+  '-DEMSESP_DEFAULT_BOARD_PROFILE="S32"'
 
-[env:ci_s3]
-; 16MB ESP32-S3 - like BBQKees S3
+[env:ci_s_16M]
+; 16MB ESP32 - no PSRAM - like some of the older BBQKees S32
 extends = espressi32_base
```

Comment:
without psram it should use the tasmota base to save ram.

Reply:
good idea. I modified it.